src/server/api/controllers/messages-controller.js

```diff
@@ -14,7 +14,9 @@ const getMessageById = async (id) => {
   try {
     const messages = await knex('messages')
       .select('messages.id as id', 'title')
-      .where({ id });
+      .where({
```

Comment:
These changes feel like a formatting error, can you format this file again or remove it from PR?